test/dune

```diff
@@ -156,3 +156,21 @@
  (name   runtest)
  (deps   (:x section.out) (:y section.out.expected))
  (action (diff? %{x} %{y})))
+
+; Test failures
+
+(rule
+ (targets failures.outputs)
+ (deps    (package mdx) failures/empty.ml)
+ (action
+  (with-outputs-to %{targets}
+   (progn
+    (system "%{bin:ocaml-mdx} test %{dep:failures/direction_not_set.md} || echo")
```

Comment:
I'm not sure `||` is very portable on windows

Reply:
It's not indeed but I don't know any work around for that. I think a `run!` action in dune would help there. We should probably open an issue about that. I even wonder if I haven't opened one already as I'm using the same trick to test PPXs' errors!

Reply:
I think it will work fine.